src/test/java/unit/in/ac/bits/protocolanalyzer/utils/BeautifyTest.java

```diff
@@ -0,0 +1,36 @@
+package unit.in.ac.bits.protocolanalyzer.utils;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import in.ac.bits.protocolanalyzer.utils.Beautify;
+
+public class BeautifyTest {	
+	@Test
+	public void testBeautify() {
+		byte bytes1[] = {0 , 16 , 2 , 15 , 4};
+		String mode = "ip4";
+		assertEquals("INVALID-ADDRESS",Beautify.beautify(bytes1, mode));
+		byte bytes2[] = {-65 , 66 , -67 , 68 };
+		assertEquals("191.66.189.68",Beautify.beautify(bytes2, mode));
```

Comment:
assertions given by hamcrest library are so much better. Please use them instead of the JUnit assertions.